src/Components/App/App.js

```diff
@@ -5,12 +5,18 @@ import Header from '../Header/Header';
 import { useEffect, useState } from 'react';
 import { getAllMovies, getSingleMovie, getMovieVideo } from '../../apiCalls'
 import ErrorHandling from '../ErrorHandling/ErrorHandling'
-import { Route, Routes, useParams } from 'react-router-dom';
+import { Route, Routes, useLocation } from 'react-router-dom';
 
 function App() {
   const [movies, setMovies] = useState({})
   const [singleMovie, setSingleMovie] = useState(null)
   const [error, setError] = useState('')
+  const location = useLocation().pathname
```

Comment:
Nice job with this side affect to update the error state to null after we navigate away from the error componenet